TEST_CASE("Input fail result") failed

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
a.out is a Catch v1.6.0 host application.
Run with -? for options

-------------------------------------------------------------------------------
Input fail result
-------------------------------------------------------------------------------
test_case.cc:7
...............................................................................

test_case.cc:29: FAILED:
  CHECK_FALSE( t.to_string() == "25:02:02" )
with expansion:
  !("25:02:02" == "25:02:02")
with message:
  If we try to read an invalid time from the stream and the failbit is set, the
  time object we're reading to should not be left in an invalid state

===============================================================================
test cases: 1 | 1 failed
assertions: 6 | 5 passed | 1 failed



failed testcases: 1
passed testcases: 26
total testcases: 27
